client/src/i18n/index.js

```diff
@@ -118,7 +118,7 @@ const factory = (options = { debug: true }) =>
             },
             sessionList: {
               title: 'Your remembered sessions',
-              fullAccess: 'Full Access',
+              edit: 'edit',
```

Comment:
```suggestion
              edit: 'Edit',
```

Reply:
![2021-11](https://user-images.githubusercontent.com/62861470/142068153-2c0d4fde-fa24-4933-aff3-063bf38d2013.png)

just noticed the second language of Polish that i would have broken so I've changed the name back to fullAccess. Need to change Pełen dostęp to Edit in polish too?

Reply:
please rename fullAccess to edit (as you did previously) and remove 'fullAccess' from polish dictionary

i18n is setup so that it will fall back to english translation :) 
if it doesn't find an english translation, it will show the key (e.g. `sessionList.fullAccess`), but it will always display *something*, so you don't have to worry about breaking stuff

Reply:
please rename fullAccess to edit (as you did previously) and remove 'fullAccess' from polish dictionary

i18n is setup so that it will fall back to english translation :)
if it doesn't find an english translation, it will show the key (e.g. sessionList.fullAccess), but it will always display something, so you don't have to worry about breaking stuff